code/run_capsule.py

```diff
@@ -1,65 +1,53 @@
 """ top level run script """
+
+import argparse
+import json
+import logging
+import os
+from datetime import datetime as dt
 from pathlib import Path
+from typing import Union
+
+import decrosstalk_roi_image as dri
 import h5py as h5
-import logging
 import numpy as np
 import paired_plane_registration as ppr
-import decrosstalk_roi_image as dri
-import json
-from aind_data_schema.core.processing import Processing
-from aind_data_schema.core.processing import DataProcess, ProcessName, PipelineProcess
-from typing import Union
-from datetime import datetime as dt
-import argparse
-import os
+from aind_data_schema.core.processing import DataProcess, ProcessName
+from aind_log_utils.log import setup_logging
 
 
-def write_output_metadata(
+def write_data_process(
     metadata: dict,
     input_fp: Union[str, Path],
     output_fp: Union[str, Path],
-    url: str,
-    start_date_time: dt,
+    unique_id: str,
+    start_time: dt,
+    end_time: dt,
 ) -> None:
     """Writes output metadata to plane processing.json
 
     Parameters
     ----------
     metadata: dict
         parameters from suite2p motion correction
-    input_fp: str
-        path to data input
+    raw_movie: str
```

Comment:
The argument is named `input_fp`. to be pydantic, while the input is indeed a movie, it is motion corrected  not "raw"

Reply:
good catch. this was a copy and paste of this function. fixed